examples/big-bang/zarf.yaml

```diff
@@ -13,88 +13,68 @@ components:
 
   - name: big-bang
     required: true
-    secretName: "private-registry"
     manifests:
       - name: big-bang-config
         kustomizations:
           - "kustomization"
     repos:
-      - https://repo1.dso.mil/platform-one/big-bang/bigbang.git@1.28.0
-      - https://repo1.dso.mil/platform-one/big-bang/apps/core/istio-controlplane.git@1.11.5-bb.2
-      - https://repo1.dso.mil/platform-one/big-bang/apps/core/istio-operator.git@1.11.5-bb.1
-      - https://repo1.dso.mil/platform-one/big-bang/apps/core/jaeger.git@2.27.1-bb.4
-      - https://repo1.dso.mil/platform-one/big-bang/apps/core/kiali.git@1.45.0-bb.3
-      - https://repo1.dso.mil/platform-one/big-bang/apps/core/cluster-auditor.git@1.2.0-bb.1
-      - https://repo1.dso.mil/platform-one/big-bang/apps/core/policy.git@3.7.0-bb.9
-      - https://repo1.dso.mil/platform-one/big-bang/apps/core/elasticsearch-kibana.git@0.6.0-bb.2
-      - https://repo1.dso.mil/platform-one/big-bang/apps/core/eck-operator.git@1.9.1-bb.4
-      - https://repo1.dso.mil/platform-one/big-bang/apps/core/fluentbit.git@0.19.19-bb.0
-      - https://repo1.dso.mil/platform-one/big-bang/apps/core/monitoring.git@30.0.1-bb.8
-      - https://repo1.dso.mil/platform-one/big-bang/apps/security-tools/twistlock.git@0.4.0-bb.0
+      - https://repo1.dso.mil/platform-one/big-bang/bigbang.git@1.33.0
+      - https://repo1.dso.mil/platform-one/big-bang/apps/core/istio-controlplane.git@1.13.2-bb.1
+      - https://repo1.dso.mil/platform-one/big-bang/apps/core/istio-operator.git@1.13.2-bb.1
+      - https://repo1.dso.mil/platform-one/big-bang/apps/core/jaeger.git@2.29.0-bb.1
+      - https://repo1.dso.mil/platform-one/big-bang/apps/core/kiali.git@1.49.0-bb.0
+      - https://repo1.dso.mil/platform-one/big-bang/apps/core/cluster-auditor.git@1.4.0-bb.2
+      - https://repo1.dso.mil/platform-one/big-bang/apps/core/policy.git@3.7.1-bb.0
+      - https://repo1.dso.mil/platform-one/big-bang/apps/sandbox/loki@3.0.1-bb.1
```

Comment:
@jeff-mccoy The read-only user is getting errors for loki and promtail because they're the only two repos listed that don't have `.git` at the end of the link/before the `@{VERSION_TAG}`.

The code for the read-only user assumes .git is in the repo string provided and since it wasn't there the strings.split() call gave the wrong repo name to add the read-only user to.

When I originally wrote this, I was assuming `.git` in the repo name was a 'required' to the schema since that's how we have it everywhere else but obviously it doesn't look like that's that a checked requirement. Should it be?

Reply:
Interesting I think we already prescribe a unique format anyway so yes it should be, we should open an issue for a new validator for git urls, excellent find!